ledger-tool/src/main.rs

```diff
@@ -1032,6 +1032,8 @@ fn main() {
 
                     println!("Creating a snapshot of slot {}", bank.slot());
                     bank.squash();
+                    bank.clean_accounts();
```

Comment:
Follow the api change of https://github.com/solana-labs/solana/pull/9507/files#diff-f5031fae39eabfef864ee62e92742a1fR233